package.json

```diff
@@ -3,11 +3,13 @@
   "name": "gitpod-web-extension",
   "version": "0.0.1",
```

Comment:
Should we bump the version?

Reply:
As I understand it, this is the NPM version of this package that does not matter because we do not publish something to NPM. Does it make sense to keep that in sync with the browser extension version?